Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dobYearSelector, "Year");
+
+        cy.get("[data-testid=phnInput]")
+            .should("be.visible", "be.enabled")
+            .and("have.class", "form-control is-valid");
+    });
+
+    it("Vaccination Card - Invalid PHN via Select Year - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn").replace(/.$/, ""));
+
+        select(dobYearSelector, "Year");
+
+        cy.get(feedbackPhnMustBeValidSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - PHN, DOB and DOV not entered via Click Enter - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackPhnIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB and DOV not entered with Invalid PHN via Click Enter - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn").replace(/.$/, ""));
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackPhnMustBeValidSelector).should("be.visible");
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB and DOV not entered via Click Enter - unauthenticated user", () => {
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB and DOV cannot be future dated - unauthenticated user", () => {
+        const d = new Date();
+        const year = d.getFullYear();
+        const monthNumber = d.getMonth(); //Maps to monthNames constant array. Index starts at 0
+        const month = d.getMonth() + 1; //0 is January but UI index starts at 1 for January
+        const nextMonth = d.getMonth() === 11 ? 1 : month + 1; //When current month is December, next month will be January
+        const day = d.getDate();
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        // Future Date
+        d.setDate(d.getDate() + 1);
+        const nextYear = d.getFullYear() === year ? year + 1 : d.getFullYear();
+        const nextDay = d.getDate();
+
+        // Test Future Year does not exist
```

Comment:
I'd recommend swapping these comments into cy.log() statements so we can see them when the test runs.

Reply:
Logging added.